scripts/zkai_attention_block_statement_bridge_gate.py

```
@@ -0,0 +1,610 @@
+    if not isinstance(value, int) or isinstance(value, bool):
+        raise AttentionBlockStatementBridgeError(f"{label} must be integer")
+    return value
+
+
+def _bool(value: Any, label: str) -> bool:
+    if not isinstance(value, bool):
+        raise AttentionBlockStatementBridgeError(f"{label} must be boolean")
+    return value
+
+
+def _commitment(value: Any, label: str) -> str:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)